### ISSUE TITLE
verify covariate_figures.Rmd 

### ISSUE THREAD

**Issue by mbnebel** [mbnebel/DeconfoundedFMRI#5]
1. Create Data/noImputation subdirectory. 
 2. Edit exclude file to exclude Data subdirectory and all of its contents.
 3. Save .RData input to Data/noImputation  
 4. See if .Rmd will knit

**Comment by lidstone** [comment]
There are no errors in the code except for one section (chunk 8, line 171) where a ‘nScans.RData’ file is loaded.

**Comment by mbnebel** [comment]
Thanks for catching this! That file tracks how many resting-state scans each participant completed; I'll share it.

**Comment by lidstone** [comment]
[Exploring.covariates.rs-fMRI.usability.and.functional.connectivity.pdf](https://github.com/mbnebel/DeconfoundedFMRI/files/8354291/Exploring.covariates.rs-fMRI.usability.and.functional.connectivity.pdf)

**Comment by lidstone** [comment]
[Exploring covariates, rs-fMRI usability, and functional connectivity.pdf](https://github.com/mbnebel/DeconfoundedFMRI/files/8374318/Exploring.covariates.rs-fMRI.usability.and.functional.connectivity.pdf)

**Comment by mbnebel** [comment]
Thanks @lidstone! I should have clarified that I need you to check the [revision branch](https://github.com/mbnebel/DeconfoundedFMRI/tree/revision)

**Comment by lidstone** [comment]
Thank you @mbnebel, the following is from the revision branch. There was one .Rmd file that was giving me some problems (‘FisherTestsRaceAndSex.Rmd’). Under the 'Data/' directory I had to create a subdirectory '/imputeMotorOverflow' and added in the imputed data from the Dropbox folder titled ‘DataWithPropensities_seed1_motoroverflowimpute.RData ’. I then changed the filename to 'DataWithPropensities_seed1.RData' and the code ran without error.

![image](https://user-images.githubusercontent.com/56608854/160731233-a949b54e-d7df-4c7b-98fc-837dde2aaea0.png)

[Exploring covariates, rs-fMRI usability, and functional connectivity.pdf](https://github.com/mbnebel/DeconfoundedFMRI/files/8375981/Exploring.covariates.rs-fMRI.usability.and.functional.connectivity.pdf)
